.github/workflows/js.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           yarn i18n:check
       - name: Run tests
         run: |
-          yarn test
+          yarn test --silent
```

Comment:
cc @harupy is this change okay? it still displays test failures, but suppresses the many warnings that we have so the logs don't take up 40k lines. i think this might have been causing some flakiness in the test runners (got a few "operation was cancelled"): 
https://github.com/mlflow/mlflow/actions/runs/7592701235/job/20682521973#step:9:37230

after adding this line i don't see that error much anymore, though i'm not 100% sure if it's the fix. even if not, at least it makes our tests readable

Reply:
looks good to me

Reply:
IMO looks good as well